models/credmark/protocols/lending/aave/aave_v2.py

```diff
@@ -1,83 +1,120 @@
-from typing import (
-    List,
-    Optional,
+from models.tmp_abi_lookup import (
+    AAVE_STABLEDEBT_ABI
 )
-
-from credmark.cmf.model import Model
-
-from credmark.cmf.model.errors import (
-    ModelRunError,
-    ModelDataError
+from credmark.dto import (
+    DTO,
+    EmptyInput,
+    IterableListGenericDTO,
 )
 
 from credmark.cmf.types import (
     Address,
     Contract,
     Token,
     Position,
-    Portfolio,
+    Portfolio
 )
 
-from credmark.cmf.types.series import BlockSeries
-
-from credmark.dto import (
-    DTO,
-    EmptyInput,
-    IterableListGenericDTO,
+from credmark.cmf.model.errors import (
+    ModelRunError,
+    ModelDataError
 )
 
-from models.tmp_abi_lookup import (
-    AAVE_V2_TOKEN_CONTRACT_ABI,
-    ERC_20_TOKEN_CONTRACT_ABI,
+from credmark.cmf.model import Model
+
+from typing import (
+    List,
+    Optional,
 )
 
 
 class AaveDebtInfo(DTO):
     token: Token
+    tokenName: str
     aToken: Token
     stableDebtToken: Token
     variableDebtToken: Token
     interestRateStrategyContract: Optional[Contract]
-    totalStableDebt: int
-    totalVariableDebt: int
-    totalDebt: int
+    supplyRate: float
+    variableBorrowRate: float
+    stableBorrowRate: float
+    totalSupply_qty: float
+    totalStableDebt_qty: float
+    totalStableDebtPrinciple_qty: float
+    totalInterest_qty: float
+    totalVariableDebt_qty: float
+    totalDebt_qty: float
+    totalLiquidity_qty: float
 
 
 class AaveDebtInfos(IterableListGenericDTO[AaveDebtInfo]):
     aaveDebtInfos: List[AaveDebtInfo]
     _iterator: str = 'aaveDebtInfos'
 
 
+# For different markets
+AAVE_LENDING_POOL_ADDRESS_PROVIDER_REGISTRY = '0x52D306e36E3B6B02c153d0266ff0f85d18BCD413'
```

Comment:
We'd like to start putting hardcoded addresses into models that look these up based on the context's Chain ID, then it can error accordingly